packages/cli/src/commands/serve.ts

```diff
@@ -66,6 +67,81 @@ class PlatformAppDevelopmentServer {
     }
 }
 
+class ThemeDevelopmentServer {
```

Comment:
Maybe we should split these `DevelopmentServer` classes out of the `serve.ts` so we can add proper Ownership on them